packages/venia-concept/src/components/Navigation/navHeader.js

```diff
@@ -18,13 +18,17 @@ class NavHeader extends Component {
     };
 
     render() {
-        const { classes, onBack, onClose, title } = this.props;
+        const { classes, onBack, onClose, title, isCategoryRoot } = this.props;
 
         return (
             <Fragment>
-                <Trigger key="backButton" action={onBack}>
-                    <Icon src={ArrowLeftIcon} />
-                </Trigger>
+                {!isCategoryRoot ? (
+                    <Trigger key="backButton" action={onBack}>
+                        <Icon src={ArrowLeftIcon} />
+                    </Trigger>
+                ) : (
+                    <div />
+                )}
```

Comment:
I'd prefer defining a `backButton` or similar variable above the JSX and using that here.

For consistency, mainly, but also because with this pattern the reader has to mentally switch from JSX to JS back to JSX and that can be avoided easily.

Reply:
@supernova-at : I have performed needed changes as per suggestion, please help to have a look into and share any other concern as well with me.
Thanks.